README.md

```diff
@@ -10,6 +10,80 @@
 
 ![freeway overview diagram](./docs/freeway.png)
 
+## Running Locally
+
+1. Install the project
+```sh
+npm i
+```
+
+2. CloudFlare Authentication
+```sh
+npx wrangler login
+```
+
+3. Get Your Account Id
+```sh
+npx wrangler whoami
+```
+
+4. Add your configs to `wrangler.toml`
+```sh
+[env.YOUR_USERNAME]
+workers_dev = true
+account_id = "YOUR_ACCOUNT_ID"
+r2_buckets = [
+  { binding = "CARPARK", bucket_name = "carpark-YOUR_USERNAME-0", preview_bucket_name = "carpark-YOUR_USERNAME-preview-0" }
+]
+
+[env.YOUR_USERNAME.vars]
+DEBUG = "true"
+FF_RATE_LIMITER_ENABLED = "false"
+CONTENT_CLAIMS_SERVICE_URL = "https://dev.claims.web3.storage"
+```
+
+If you want to enable the Rate Limiter and KV add the following too:
+```sh
+[[env.YOUR_USERNAME.unsafe.bindings]]
+name = "RATE_LIMITER"
+type = "ratelimit"
+namespace_id = "0"
+simple = { limit = 100, period = 60 }
+
+[[env.YOUR_USERNAME.kv_namespaces]]
+binding = "AUTH_TOKEN_METADATA"
+id = "YOUR_TOKEN" //FIXME how to obtain this token?
+```
+
+5. Start local server
+```sh
+npx wrangler dev -e YOUR_USERNAME
```

Comment:
I think having them in the Readme should be fine for now. That `dev` script starts the worker in dev mode (but that was for miniflare v2), I've updated it to miniflare v3 and I guess we can remove that `dev` script for now because they recommend using `npx wrangler dev -e <profile>` instead.